### PR TITLE
Added settings button and updated styles

### PR DESCRIPTION
**Pull request by Copermaine**

### CODE REVIEW

File: .env.copy [hunk 1]
```diff
@@ -1,7 +0,0 @@
-DEBUG_AUTH=""
```
Comment (markvandal) [inline .env.copy:1]: Please restore the file

File: .gitignore [hunk 2]
```diff
@@ -18,4 +18,4 @@ web-build/
 
 # Configs
 .env
-.env.copy*
```
Comment (markvandal) [inline .gitignore:21]: Please restore the entery

File: components/header.js [hunk 3]
```diff
@@ -1,23 +1,28 @@
...
 
-import { Image } from 'react-native'
 
-import { withGalio, NavBar, Text, Block } from 'galio-framework'
+export const Header = withGalio(({navigation, scene, theme, styles}) => {
+    const {options} = scene.descriptor
+    return <Block flex center>
+            <Block bottom style={styles.header_settings}>
+                <Button onlyIcon icon="gear" iconFamily="EvilIcons"
+                        iconSize={25}
```
Comment (markvandal) [inline components/header.js:13]: White spaces should be double space per tab. Please fix.

**Review by markvandal — CHANGES_REQUESTED** [review]
1. Please fix white spaces in your IDE (I'd recommend to use VS code with it's autoformatting configured for double space per tab).
2. Restore missing files.

**Comment by Copermaine** [comment]
Double space per tab fixed.

**Comment by Copermaine** [comment]
Restore env

File: .gitignore (hunk 2 above)
Comment (jackmb1) [reply .gitignore:21]: and probably make sense to ignore `/.idea/` folder as it's your env specific stuff

File: package.json [hunk 4]
```diff
@@ -39,7 +39,8 @@
     "react-native-web": "~0.13.12",
     "react-redux": "^7.2.2",
     "reconnecting-websocket": "^4.4.0",
-    "uuid": "^3.4.0"
+    "uuid": "^3.4.0",
```
Comment (jackmb1) [inline package.json:42]: has `package-lock.json` been changed as well? https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json

File: .gitignore (hunk 2 above)
Comment (markvandal) [reply .gitignore:21]: idea is still here )

**Review by markvandal — CHANGES_REQUESTED** [review]

**Comment by Copermaine** [comment]
Close

**Pull request closed without merging**